tests/test_jump_cas22.py

```diff
@@ -24,7 +24,7 @@
 READ_NOISE = np.float32(5)
 
 # Set a value for jumps which makes them obvious relative to the normal flux
-JUMP_VALUE = 10_000
+JUMP_VALUE = 1_000
```

Comment:
Looks good to me.  I wasn't sure if this one was intentional.

Reply:
This had no visible effect on the results